playground/backend/internal/validators/java_validators_test.go

```diff
@@ -63,14 +62,67 @@ func TestCheckIsUnitTests(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := CheckIsUnitTestJava(tt.args.args...)
+			got, err := checkIsUnitTestJava(tt.args.args...)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("checkIsUnitTestJava error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if got != tt.want {
+				t.Errorf("checkIsUnitTestJava got = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestCheckIsKataJava(t *testing.T) {
+	testValidatorArgs := getValidatorsArgs(javaUnitTestFilePath, javaUnitTestPattern)
+	validatorArgs := getValidatorsArgs(javaCodePath, javaUnitTestPattern)
```

Comment:
Shouldn't there be `javaKataFilePath` instead of `javaUnitTestFilePath ` and `javaKatasPattern` instead of `javaUnitTestPattern`?

Reply:
Done.